massa-client/src/cmds.rs

```diff
@@ -482,6 +488,21 @@ impl Command {
                 )
                 .await
             }
+            Command::when_episode_ends => {
+                if let (Some(end), version) = match client.public.get_status().await {
+                    Ok(node_status) => (node_status.algo_config.end_timestamp, node_status.version),
+                    Err(e) => bail!("RpcError: {}", e),
+                } {
+                    let (days, hours, mins, secs) =
+                        end.saturating_sub(UTime::now(0)?).days_hours_mins_secs()?; // TODO set compensation millis
```

Comment:
Can we find easily the compensation in the client? Or do we need to update the node API?

Reply:
I added the compensation millis to the api.

So @qdrn @Noxhaa get_status has one more field.

Reply:
We should NOT send the compensation_millis from the node to the client !
In many cases, the client won't even be running on the same computer as the node.
Use compensation_millis = 0 in the client

Reply:
So the now in the client that is used to process data won't be the same now as in the node that produced that data ?

Reply:
yup, that's ok, especially if the client is running on a different computer

Reply:
Done !

Reply:
Done